hasher-matcher-actioner/hmalib/lambdas/fetcher.py

```diff
@@ -85,6 +85,17 @@ def get(cls):
         )
 
 
+def is_int(int_string):
```

Comment:
nit: maybe call this `is_intish`

Reply:
We cast it to an int directly in this function and a few lines after the call. What values are you thinking of as you suggested this change?